tests/PHPUnit/Unit/Period/RangeTest.php

```diff
@@ -182,6 +182,16 @@ public function testRangeComma3_EndDateIncludesToday()
         $this->assertEquals('2008-01-03', $subPeriods[2]->toString());
     }
 
+    // test range date1,date2
+    public function testRangeComma4_EndDateIncludesTodayWithTimezone()
```

Comment:
can we also add some tests for other timezones that are say +10 hours and -10 hours?

Reply:
and also check `getEndDate()->toDatetime()` or so?

Reply:
add some more tests